scripts/zkai_d128_risc0_statement_receipt_gate.py

```
@@ -0,0 +1,618 @@
+    }
+
+
+def rzup_components() -> dict[str, str]:
+    if shutil.which("rzup") is None:
+        return {}
+    result = subprocess.run(["rzup", "show"], check=False, text=True, capture_output=True, timeout=20, cwd=ROOT)
+    components: dict[str, str] = {}
+    current_name: str | None = None
+    for raw_line in result.stdout.splitlines():
+        line = raw_line.strip()
+        if not line or line.startswith("-") or line.startswith("Installed components") or line.startswith("rzup home"):
```

Ruff (0.15.12)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)